model-optimizer/mo/utils/cli_parser.py

```diff
@@ -325,6 +325,9 @@ def get_common_cli_parser(parser: argparse.ArgumentParser = None):
     common_group.add_argument('--transformations_config',
                           help='Use the configuration file with transformations description.',
                           action=CanonicalizePathCheckExistenceAction)
+    common_group.add_argument('--use_fallback',
```

Comment:
Please, change the command line name for something more meaningful. Right now it is not clear what kind of fallback we mean.

Reply:
Done.